### PR TITLE
Support for `meta.json` file

### PR DESCRIPTION
**Pull request by rbonichon**
Add symbol to debugging/info output

### CODE REVIEW

**Review by dannywillems — CHANGES_REQUESTED** [review]
LGTM, but would be nice to have a meta.json file example and see if it is parsed correctly with a test

File: optimism/src/cannon.rs [hunk 1]
```diff
@@ -152,3 +137,165 @@ impl Start {
...
+    fn test_find_address_symbol() {
+        let meta = deserialize_meta_sample();
+
+        assert_eq!(
+            meta.find_address_symbol(69633),
+            Some("internal/cpu.processOptions".to_string())
+        );
+        assert_eq!(
+            meta.find_address_symbol(69632),
+            Some("internal/cpu.processOptions".to_string())
```
Comment (dannywillems) [inline optimism/src/cannon.rs:297]:
```suggestion
            Some("runtime.text".to_string())

```
?

Comment (dannywillems) [inline optimism/src/cannon.rs:297]: Duplicated?

File: optimism/src/cannon.rs [hunk 2]
```diff
@@ -152,3 +137,165 @@ impl Start {
...
+                    }
+                }
+            },
+        );
+
+        match res {
+            Ok(idx) => Some(self.symbols[idx].name.to_string()),
+            Err(_) => None,
+        }
+        // for e in self.symbols.iter() {
```
Comment (dannywillems) [inline optimism/src/cannon.rs:180]: Old?

**Review by dannywillems — APPROVED** [review]
LGTM modulo last comments, but Clippy is grumpy.

File: optimism/src/cannon.rs (hunk 1 above)
Comment (rbonichon) [reply optimism/src/cannon.rs:297]: No. Not testing the same value.

File: optimism/src/cannon.rs (hunk 2 above)
Comment (rbonichon) [reply optimism/src/cannon.rs]: Yep. removed

**Pull request merged**